cfme/tests/containers/test_containers_properties.py

```diff
@@ -17,106 +23,181 @@
-        obj = Pod(name, provider)
-        assert getattr(obj.summary.properties, rel).text_value
-
-# CMP-9877
-
-
-@pytest.mark.parametrize('rel',
-                         ['name',
-                          'creation_timestamp',
-                          'resource_version',
-                          'host_name'
-                          ])
-def test_routes_properties_rel(provider, rel):
-    """ Properties table fields tests - Containers Routes' summary page
-    This test verifies the fields of the Properties table in Containers Routes'
-    details menu
-    Steps:
-    Containers -- > Containers Routes
-    Loop through each Route object in the table and check validity of
-    the fields in the Properties table
-    """
-    navigate_to(Route, 'All')
-    ui_routes = [r.name.text for r in list_tbl_routes.rows()]
-
-    for name in ui_routes:
-        obj = Route(name, provider)
-        assert getattr(obj.summary.properties, rel).text_value
-
-# CMP-9867
-
-
-@pytest.mark.parametrize('rel',
-                         ['name',
-                          'creation_timestamp',
-                          'resource_version'
-                          ])
-def test_projects_properties_rel(provider, rel):
-    """ Properties table fields tests - Containers Projects' summary page
-    This test verifies the fields of the Properties table in Containers Projects'
-    details menu
-    Steps:
-    Containers -- > Containers Projects
-    Loop through each Project object in the table and check validity of
-    the fields in the Properties table
-    """
-    navigate_to(Project, 'All')
-    ui_projects = [r.name.text for r in list_tbl_projects.rows()]
-
-    for name in ui_projects:
-        obj = Project(name, provider)
-        assert getattr(obj.summary.properties, rel).text_value
-
-# CMP-9884
-
-
-@pytest.mark.parametrize('rel',
-                         ['name',
-                          'creation_timestamp',
-                          'resource_version',
-                          'session_affinity',
-                          'type',
-                          'portal_ip'
-                          ])
-def test_services_properties_rel(provider, rel):
-    """ Properties table fields tests - Containers Services' summary page
-    This test verifies the fields of the Properties table in Containers Services'
-    details menu
-    Steps:
-    Containers -- > Containers Services
-    Loop through each Service object in the table and check validity of
-    the fields in the Properties table
-    """
-    navigate_to(Service, 'All')
-    ui_services = [r.name.text for r in list_tbl_services.rows()]
-
-    for name in ui_services:
-        obj = Service(name, provider)
-        assert getattr(obj.summary.properties, rel).text_value
+
+def rand_choice(lst, cnt):
+    """Choosing <cnt> random items from <lst>"""
+    shuffle(lst)
+    return [lst.pop(-1) for _ in xrange(min((cnt, len(lst))))]
+
+
+def align_types(ui_val, api_val):
```

Comment:
Well, I'll change to use literal_eval for the bool string cases, but there are still cases that <value> comes from the API as (<value>, ) and similar cases...
I agree about that this is not a good place for this, I can fix the mgmtsystem instead but this will effect all the framework and will require much more reviewing time and testing.
So - fix the API?